test/startRoundUnitTest.js

```diff
@@ -45,7 +46,8 @@ contract('ROSCA startRound Unit Test', function(accounts) {
     }));
 
     it("Throws when calling startRound before roundStartTime (including round = 0)", co(function* () {
```

Comment:
the description and test case need to be updated. Maybe `Throws when calling startRound during round 1`?

Reply:
i think we just need to get rid of `(including round 0)` because it goes through all the rounds